stackrox/base/horizontalpodautoscalers/HorizontalPodAutoscaler.yaml

```diff
@@ -0,0 +1,23 @@
+# Source: stackrox-central-services/templates/02-scanner-08-hpa.yaml
+apiVersion: autoscaling/v1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: scanner
+  namespace: stackrox
+  labels:
+    app.kubernetes.io/component: scanner
+    app.kubernetes.io/instance: stackrox-central-services
+    app.kubernetes.io/managed-by: Helm
+    app.kubernetes.io/name: stackrox
+    app.kubernetes.io/part-of: stackrox-central-services
+    meta.helm.sh/release-name: stackrox-central-services
+    meta.helm.sh/release-namespace: stackrox
+    owner: stackrox
+spec:
+  minReplicas: 2
+  maxReplicas: 5
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: scanner
+  targetCPUUtilizationPercentage: 150
```

Comment:
Are we sure this should be 150?

based off this [thread I found on stack-overflow](https://stackoverflow.com/questions/41643709/kubernetes-autoscaler-targetcpuutilizationpercentage) `The CPU utilization is the average CPU usage of a all pods in a deployment across the last minute divided by the requested CPU of this deployment.` It would seem to me that we would want to consider scaling our pods around the time they hit their requested resources, rather than once they exceed it by 50% would be the way to go. That would make sense to me seeing as the `limits` and `requests` for cpu in the stackrox namespace are the exact same, see the [x-small `resourceQuota`](https://github.com/operate-first/apps/blob/master/cluster-scope/components/resourcequotas/x-small/resourcequotas.yaml) as this is the one used in the stackrox namespace.